A3A/addons/core/Templates/Templates.hpp

```diff
@@ -474,7 +474,7 @@ class Templates
     class VN_Civ : VN_Base
     {
         side = "Civ";
-        flagTexture = "\vn\objects_f_vietnam\flags\vn_flag_01_lao_co.paa";
+        flagTexture = "x\A3A\addons\core\Pictures\Markers\vn_flag_01_vn_co.paa";
```

Comment:
Any particular reason against using QPATH instead?
```suggestion
        flagTexture = QPATHTOFOLDER(Pictures\Markers\vn_flag_01_vn_co.paa);
```